### PR TITLE
Anti-CSRF token addition

### PR DESCRIPTION
**Pull request by fourcube**
Adds Rails' "authenticity_token" anti-CSRF parameter to the configuration defaults.

### CODE REVIEW

**Comment by thc202** [comment]
Might worth adding to AntiCsrfParam.DEFAULT_TOKENS_NAMES as well.

**Comment by fourcube** [comment]
Thanks! I've added it to the AntiCsrfParam.DEFAULT_TOKEN_NAMES and updated my commit accordingly.

File: src/org/zaproxy/zap/extension/anticsrf/AntiCsrfParam.java [hunk 1]
```diff
@@ -43,7 +43,7 @@
     private static final String CONFIRM_REMOVE_TOKEN_KEY = ANTI_CSRF_BASE_KEY + ".confirmRemoveToken";
     
     private static final String[] DEFAULT_TOKENS_NAMES = { "anticsrf",
-            "CSRFToken", "__RequestVerificationToken", "csrfmiddlewaretoken" };
+            "CSRFToken", "__RequestVerificationToken", "csrfmiddlewaretokena", "authenticity_token" };
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/anticsrf/AntiCsrfParam.java]: There's a typo, an 'a' was added to the token "csrfmiddlewaretoken".

**Comment by fourcube** [comment]
I've corrected the mistake and updated my commit.

**Comment by thc202** [comment]
Thank you for the contribution!

**Comment by thc202** [comment]
How would you like to be credited? 
https://github.com/zaproxy/zap-core-help/wiki/HelpCredits

**Comment by fourcube** [comment]
@thc202 just by Chris Grieger

**Comment by thc202** [comment]
Done in zaproxy/zap-core-help@75f7a8757bf8c67b0ae69c37e5f2f9e30ca1fa69

**Pull request merged**